cilium-health/launch/endpoint.go

```diff
@@ -207,12 +207,15 @@ func LaunchAsEndpoint(owner endpoint.Owner, hostAddressing *models.NodeAddressin
 		log.WithError(err).Fatal("Error while configuring cilium-health routes")
 	}
 
-	// Propagate health IPs to all other nodes
 	if k8s.IsEnabled() {
+		// Propagate health IPs to all other nodes
 		err := k8s.AnnotateNode(k8s.Client(), node.GetName(), nil, nil, ip4, ip6)
 		if err != nil {
 			log.WithError(err).Fatal("Cannot annotate node CIDR range data")
 		}
+
+		// Set a k8s namespace so a CiliumEndpoint object can be created
+		ep.SetK8sNamespace("kube-system")
```

Comment:
Transform this into a constant please :-)

Reply:
good point